calculator.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "icalculator.h"
+
+namespace dev {
+
+class IToken {
+public:
+    IToken(const std::string& token);
+private:
+    const std::string& token_;
+};
+
+class Operand : public IToken {
+public:
+    Operand(const std::string& token);
+    double value() const;
+private:
+    double value_;
+};
+
+class Operator : public IToken {
+public:
+    Operator(const std::string& token);
+    char value() const;
```

Comment:
Operand::value and Operator::value looks VERY similar. But in fact this is totally different functions. 
Suggest to replace Operator::value -> Operator::calculate(op1, op2)

Reply:
Done.